boards/thingy52/Makefile.include

```diff
@@ -0,0 +1,12 @@
+# CPU configuration for the Thingy:52
+export CPU = nrf52
+CPU_MODEL = nrf52832xxaa
+
+# set default port depending on operating system
+PORT_LINUX ?= /dev/ttyUSB0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.SLAB_USBtoUART*)))
+include $(RIOTMAKE)/tools/serial.inc.mk
+
+# setup JLink for flashing
+export JLINK_DEVICE := nrf52
+include $(RIOTMAKE)/tools/jlink.inc.mk
```

Comment:
You should rather `include $(RIOTBOARD)/common/nrf52xxxdk/Makefile.include` here (compare #8035)

Reply:
nope. This board is not based on the common `nrf52xxxxdk` configuration, and it uses a (slightly) different configuration than those...

Reply:
but I will have a look into opimizing the shared parts in a follow up.

Reply:
At least `CPU`,  `serial.inc.mk`, `JLINK_DEVICE`, `jlink.inc.mk` are equal. Also the `Makefile.features` is nealry the same. The only difference is missing `periph_spi` for the Thingy. Doesn't this board support SPI?

Reply:
> but I will have a look into opimizing the shared parts in a follow up.

I think it's really not much to do and can be directly done in here. Would you mind adapt here before merging?

Reply:
ok, will do.